plugin/storage/es/spanstore/reader.go

```diff
@@ -85,18 +85,20 @@ var (
 
 // SpanReader can query for and load traces from ElasticSearch
 type SpanReader struct {
-	ctx    context.Context
-	client es.Client
-	logger *zap.Logger
+	ctx          context.Context
+	client       es.Client
+	logger       *zap.Logger
+	spanLookback int // Number of previous days to lookback for spans.
 }
 
 // NewSpanReader returns a new SpanReader.
-func NewSpanReader(client es.Client, logger *zap.Logger) *SpanReader {
+func NewSpanReader(client es.Client, logger *zap.Logger, spanTTL int) *SpanReader {
```

Comment:
* `s/int/time.Duration/`
 * rethink (and document) the name. TTL usually refers to something that _this_ service manages, but in fact it's managed externally by purging indices. "lookback" is weird. Maybe "maxSpanAge".